crates/ibc/src/core/ics03_connection/error.rs

```diff
@@ -17,8 +16,6 @@ pub enum ConnectionError {
     Client(client_error::ClientError),
     /// invalid connection state: expected `{expected}`, actual `{actual}`
     InvalidState { expected: String, actual: String },
-    /// invalid connection end error: `{0}`
-    InvalidConnectionEnd(ProtoError),
```

Comment:
Why did we have this? Looks like, it's not being used anywhere in the codebase.

Reply:
That's right, it's a stale variant. Not being used anymore.